_topics/en/static-channel-backups.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Static channel backups
+
+## Optional.  Shorter name to use for reference style links e.g., "foo"
+## will allow using the link [topic foo][].  Not case sensitive
+# shortname: foo
+
+## Optional.  An entry will be added to the topics index for each alias
+#aliases:
+#  - Foo
+
+## Required.  At least one category to which this topic belongs.  See
+## schema for options
+categories:
+  - Lightning Network
+
+## Optional.  Produces a Markdown link with either "[title][]" or
+## "[title](link)"
+primary_sources:
+    - title: BOLT2
+
+## Optional.  Each entry requires "title", "url", and "date".  May also use "feature:
+## true" to bold entry
+optech_mentions:
+  - title: "C-Lightning #1854 partly implements option_data_loss_protect"
+    url: /en/newsletters/2018/08/28/#c-lightning-1854
+
+  - title: "LND #2370 now updates a channel.backup file each time a new channel is opened or closed"
+    url: /en/newsletters/2019/01/29/#lnd-2370
+
+  - title: Closing lost channels with only a BIP32 seed and option_data_loss_protect
+    url: /en/newsletters/2021/05/05/#closing-lost-channels-with-only-a-bip32-seed
+
+## Optional.  Same format as "primary_sources" above
+# see_also:
+#   - title:
+#     link:
+
+## Required.  Use Markdown formatting.  Only one paragraph.  No links allowed.
+## Should be less than 500 characters
+excerpt: >
+  **Static channel backups** are backup files that only need to be
+  updated when the node opens or closes a new channel.  In case of data
+  loss, they allow the node to get the latest channel state from their
+  remote peer.
+
+
+---
+The basic idea is that a node that has potentially lost some of its
+state can encourage its peer to initiate a channel close. Since the peer
+still has the most recent state, it should close the channel using that
```

Comment:
"Since the peer should have the most recent state" - it's possible that both peers fail at the same time!